css-grid-classes/styles.css

```diff
@@ -0,0 +1,38 @@
+.container {
```

Comment:
the only thing missing here is the max-width property from the instructions. Please update!

Reply:
![Screen Shot 2020-11-12 at 8 15 34 PM](https://user-images.githubusercontent.com/60412330/99028757-790f5e00-2525-11eb-9fb4-338c63a2afb8.png)

I made the updates to the max-width property. Let me know how everything looks!